index.html

```diff
@@ -4,464 +4,524 @@
+                  Anywhere
+                </span>
+                <p class="card-content__button-text black-txt">$10k - $25k</p>
               </div>
             </div>
 
             <!-- Third card -->
-            <div class="first-section__third-card cards">
-              <div class="cards__content-container">
-                <!-- Cards content top -->
-
-                <div class="cards-content__top">
-                  <div class="cards-content__left-side">
-                    <div class="cards-content__cards-circle">
-                      <img src="./src/icons/indeed-2.svg" alt="indeed icon" />
-                    </div>
-                    <p class="cards-content__title black-txt">Indeed</p>
+            <div class="job-posts__third-card card">
+              <!-- card content top -->
+
+              <div class="card-content__top">
+                <div class="card-content__left-side">
+                  <div class="card-content__card-circle">
+                    <img src="./src/icons/indeed-2.svg" alt="indeed icon" />
+                  </div>
+                  <span class="card-content__title black-txt">Indeed</span>
+                </div>
+                <div class="card-content__right-side">
+                  <div class="right-side__first-circle">
+                    <img src="./src/icons/avatar-7.svg" alt="first avatar" />
+                  </div>
+                  <div class="right-side__second-circle">
+                    <img src="./src/icons/avatar-6.svg" alt="second avatar" />
                   </div>
-                  <div class="cards-content__right-side">
-                    <div class="right-side__first-circle">
-                      <img src="./src/icons/avatar-7.svg" alt="first avatar" />
-                    </div>
-                    <div class="right-side__second-circle">
-                      <img src="./src/icons/avatar-6.svg" alt="second avatar" />
-                    </div>
-                    <div class="right-side__third-circle">
-                      <p class="avatar-text">5+</p>
-                      <img src="./src/icons/avatar-3.svg" alt="third avatar" />
-                    </div>
+                  <div class="right-side__third-circle">
+                    <p class="avatar-text">5+</p>
+                    <img src="./src/icons/avatar-3.svg" alt="third avatar" />
                   </div>
                 </div>
+              </div>
 
-                <!-- Cards content middle -->
-                <div class="cards-content__middle">
-                  <h2 class="cards-content__middle_title tltle">
-                    Project Manager
-                  </h2>
-                  <p class="cards-content__middle_subtitle">
-                    Full Time - Singapore
-                  </p>
-                </div>
+              <!-- card content middle -->
+              <div class="card-content__middle">
+                <p class="card-content__middle_title black-txt">
```

Comment:
```suggestion
                <p class="middle-title black-txt">
```

We use an underscore to mark modifiers, but "_title" is not a modifier.

Reply:
Renamed class